Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:2.7
```

Comment:
Can you add a comment in the top of this file explaining this specifies a container for just the daemon, where the daemon python sources are put under /opt/spinnaker-monitoring/bin? It might be subtle.

Also add somewhere instructions reminding that the spinnaker-monitoring.yml and sources/*.yml configuration files are still needed in the container to make it functional.